app/assets/stylesheets/content/_widget_box.sass

```diff
@@ -120,6 +108,36 @@ $widget-box--enumeration-width: 20px
       margin-left: auto
       margin-right: auto
 
+.widget-box--header
+  font-weight: bold
+  font-size: 1.25rem
+  display: flex
+
+  .icon:before,
+  .icon-context:before
+    display: inline-block
+    padding-top: 0
+    vertical-align: middle
+
+.widget-box--header-title
+  vertical-align: middle
+  margin-bottom: 0
+  flex-grow: 1
+  align-self: center
+
+  .widget-box--header-icon + &
+    padding-left: 5px
+
+  // Ensure style for read-only widget headers
+  .editable-toolbar-title--fixed
+    color: #5F5F5F
+    font-size: 20px
+    font-weight: normal
+
+.widget-box--header-icon
+  align-self: center
+  margin-top: -5px
```

Comment:
LGTM mostly, but this margin-top results in visible offset in my case. Depending on what icon one is looking it, margin-top: -2px or none at all looks better on my machine. Maybe it's a platform issue

Reply:
I'll try to come up with a non 'magic number' approach if possible.

Reply:
@oliverguenther I now use an approach which is based on line height (oriented on the title element right next to the icon) so all browsers should then center align the icon

**BUT** as our icons do not have a unified dimension (some have some pixel space at the bottom, some overflow their containers) it will not be possible to find a solution that fits them all. We would have to regenerate the icons without those caveats to find one.